docs/error-reporting/platform-integrations/unity/setup.md

```diff
@@ -30,10 +30,14 @@ The Backtrace Unity SDK reports on the following types of errors:
 | Supported Platforms | Supported Systems                                                                     |
 | ------------------- | ------------------------------------------------------------------------------------- |
 | Mobile              | Android, iOS                                                                          |
-| PC                  | Windows, MacOS                                                                        |
+| PC                  | Windows, MacOS*                                                                       |
 | Web                 | WebGL                                                                                 |
 | Game Consoles       | PlayStation 4, PlayStation 5, Xbox One, Xbox Series X, Nintendo Switch.               |
 
+:::note
+Native Crashes on MacOS are not supported via backtrace-unity.
```

Comment:
we have to ask the same not to WebGL and Game Consoles

Reply:
There is already a note that says: 

> Native Crashes on PlayStation and Nintendo Switch are captured via Backtrace Data Sources, which connect to the vendor provided crash reporting server. Native Crashes on Xbox can be captured by installing an additional dynamic link library (DLL) for Backtrace-Unity. Reach out to your Xbox partner manager to verify developer status with Sauce Labs Backtrace.

but I suppose we should add the WebGL one